docs/resources/dashboard_json.md

```diff
@@ -115,6 +115,142 @@ EOF
 }
 ```
 
+## Reading from external file
+Following is the example dashboard with sections and parameters from external file.
```

Comment:
Below is the example dashboard with sections and parameters from an external file.